hs_core/views/resource_folder_hierarchy.py

```diff
@@ -172,7 +172,9 @@ def data_store_folder_unzip(request):
     try:
         unzip_file(user, res_id, zip_with_rel_path, bool_remove_original)
     except SessionException as ex:

```

Comment:
That is a good question, I am not sure exactly how to do it yet, and I need to investigate a bit to come up with the best approach, which is to only popup a message when there are files with the same names to override and then act accordingly. If there are no conflict files with the same names, it should proceed without asking users. I believe this should be doable to check whether there are name conflict issues before unzipping without relying on iRODS operation. In any case, I think this interim workaround would go away in the subsequent implementation.